index.html

```diff
@@ -279,11 +281,15 @@ <h2 id="h-tag" class="desktop-multipost">Uber Navigation</h2>
             <li class="small-langli">javascript</li>
           </ul>
           <div>
-            <button type="button" class="seeproject desktop-multipost active">See Project</button>
+            <button id="desktop4" onclick="displayModal(3)" type="button" class="seeproject desktop-multipost active">See Project</button>
           </div>
         </div>
       </div>
     </section>
+    <section>
+      <div id="modal" class="workSection modal"></div>
```

Comment:
- [ Required ] It would be better if your design tries as much as possible to match the design provided on the `Figma` template. On the Figma template when the modal is open, it's very difficult to see the elements in the background on the page because of the `blur` effect on the modal background. In your design the page elements in the design are quite visible, kindly fix this.

## Figma template
![Screenshot from 2023-03-10 11-32-07](https://user-images.githubusercontent.com/16392046/224265913-ab169069-0fc9-4204-8f6d-c3b38ddab257.png)

## Your design
![Screenshot from 2023-03-10 11-32-28](https://user-images.githubusercontent.com/16392046/224265734-34ac4b18-7e6b-47ff-8a33-efccf12f051e.png)